src/Tracy/BlueScreen/assets/content.phtml

```diff
@@ -298,12 +298,7 @@ $code = $exception->getCode() ? ' #' . $exception->getCode() : '';
 
 		<div class="tracy-collapsed inner">
 			<h3>URL</h3>
-			<p>
```

Comment:
Please can you add `rel="noreferrer noopener"` after `target="_blank"`?

Reply:
Added.